src/tools/crushtool.cc

```diff
@@ -128,38 +128,39 @@ void usage()
   cout << "   [--outfn|-o outfile]\n";
   cout << "                         specify output for for (de)compilation\n";
   cout << "   --compile|-c map.txt  compile a map from source\n";
-  cout << "   --enable-unsafe-tunables compile with unsafe tunables\n";
+  cout << "   --enable_unsafe_tunables\n"
+  cout << "                         compile with unsafe tunables\n";
   cout << "   --build --num_osds N layer1 ...\n";
   cout << "                         build a new map, where each 'layer' is\n";
-  cout << "                           'name (uniform|straw|list|tree) size'\n";
+  cout << "                         'name (uniform|straw|list|tree) size'\n";
   cout << "\n";
   cout << "Options for the tunables adjustments stage\n";
   cout << "\n";
-  cout << "   --set-choose-local-tries N\n";
+  cout << "   --set_choose_local_tries N\n";
   cout << "                         set choose local retries before re-descent\n";
-  cout << "   --set-choose-local-fallback-tries N\n";
+  cout << "   --set_choose_local_fallback_tries N\n";
   cout << "                         set choose local retries using fallback\n";
   cout << "                         permutation before re-descent\n";
-  cout << "   --set-choose-total-tries N\n";
+  cout << "   --set_choose_total_tries N\n";
   cout << "                         set choose total descent attempts\n";
-  cout << "   --set-chooseleaf-descend-once <0|1>\n";
+  cout << "   --set_chooseleaf_descend_once <0|1>\n";
   cout << "                         set chooseleaf to (not) retry the recursive descent\n";
-  cout << "   --set-chooseleaf-vary-r <0|1>\n";
+  cout << "   --set_chooseleaf_vary_r <0|1>\n";
   cout << "                         set chooseleaf to (not) vary r based on parent\n";
   cout << "   --set-chooseleaf-stable <0|1>\n";
   cout << "                         set chooseleaf firstn to (not) return stable results\n";
   cout << "\n";
   cout << "Options for the modifications stage\n";
   cout << "\n";
-  cout << "   -i mapfn --add-item id weight name [--loc type name ...]\n";
+  cout << "   -i mapfn --add_item id weight name [--loc type name ...]\n";
   cout << "                         insert an item into the hierarchy at the\n";
   cout << "                         given location\n";
-  cout << "   -i mapfn --update-item id weight name [--loc type name ...]\n";
+  cout << "   -i mapfn --update_item id weight name [--loc type name ...]\n";
   cout << "                         insert or move an item into the hierarchy at the\n";
   cout << "                         given location\n";
-  cout << "   -i mapfn --remove-item name\n"
```

Comment:
@dillaman Besides, some of the tips conflict with the implementation. Take "--remove-item" for example, the implementation(check line 415 below here) really considers "--remove_item" as a match, although we can rightfully process both internally.